nio/properties/util/parser.py

```diff
@@ -76,6 +76,7 @@ def _unescape(self, match):
     def _transform_attr(self, match):
         tok = match.group(0).split('$')
         result = tok[0]
+        # add extra check for len(tok) for python37 compatibility
```

Comment:
I would rather see the check here than a todo, or neither if out of scope for this PR.

Reply:
This wasn't meant to be a TODO actually, it was a leftover comment for code that got moved to #142 so it will get merged and fixed in that one. Good catch

Reply:
:+1: